common/src/main/java/com/brandwatch/minibcr/common/domain/Query.java

```diff
@@ -0,0 +1,40 @@
+package com.brandwatch.minibcr.common.domain;
+
+public class Query {
+
+    private long id;
+
+    private String definition;
+
+    public Query(long id, String definition) {
+        this.id = id;
+        this.definition = definition;
+    }
+
+    public Query(String definition) {
+        this.definition = definition;
+    }
+
+    public Query(){}
```

Comment:
Any reason why you have a blank constructor? While valid, it's not something we typically do.

Reply:
So this was mandatory when I used JPA instead of JDBC and I assumed its the same so I left it. I'll get rid of it now 😅

Reply:
Actually, I need it for the rowMapper because I create a Query object without any parameters, but that could be avoided as well